docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/api/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Managing Data Products via the API"
+sidebar_label: "Using the API"
+sidebar_position: 2
+sidebar_custom_props:
+  offerings:
+    - enterprise
+---
+
+As well as managing [data product](/docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/index.md) through the Snowplow BDP Console, Snowplow BDP customers can also manage them programmatically through an API.
+
+This functionality is key to automating any existing process you may have, including workflows in version control systems like GitHub.
+
+Partnered with other tools like our [CI tool](/docs/managing-data-quality/testing-and-qa-workflows/using-the-data-structures-ci-tool-for-data-quality/index.md) and / or [Snowplow Micro](/docs/testing-debugging/snowplow-micro/what-is-micro/index.md), it's possible to have a very robust and automated data structure workflow that ensures data quality upstream of data hitting your pipeline.
+
+## Getting started
+
+You can have a look at and interact with all available endpoints in the [API documentation](https://console.snowplowanalytics.com/api/msc/v1/docs).
+
+### Authorizing in the API documentation
+
+To be able to post sample requests in the documentation you need to click the `Authorize` button at the top of the document and authorize with your token. The value for the token field in each individual requests is overwritten by this authorization.
+
+The endpoints focus on the main operations in the workflow around:
+
+1. Retrieving existing data products and their event specifications, also known as tracking scenarios in the current v1 API version
+2. Creating or editing new or existing data products
+3. Seeing the data product history
+4. Managing user subscriptions
+
+Each request will need to include your company's `organizationID` which is a UUID that can be retrieved from the URL immediately following the .com when visiting console:
+
+![](images/orgID.png)
+
+```mdx-code-block
+import GetConsoleApiKey from "@site/docs/reusable/get-console-api-key/_index.md"
+
+<GetConsoleApiKey/>
+```
+
+## Retrieving Information about Data Products
+
+The following `GET` requests are designed to allow you to access information about data products. These data products could be stored in either your development environment registry or your production environment registry.
+
+### Retrieve a List of All Data Products
+
+To retrieve a comprehensive list of all data products in your organization, you can use the following GET request:
+
+`**GET** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1`
+
+Query parameter `organizationId` is required.
+
+### Retrieving Information about a Specific Data Product
```

Comment:
When retrieving a data product, it could also contain an array field `data[].tracking_scenarios` (aka Event Spec) that will contain the id and url of the associated tracking scenarios. E.g:
```
"data": [
      ...
      "trackingScenarios": [
        {
          "id": "d1336abc-1b60-46f7-be2d-2105f2daf283",
          "url": "https://next.console.snowplowanalytics.com/api/msc/v1/organizations/f51dada7-4f11-4b6a-bbbd-2cf6a3673035/tracking-scenarios/v1/d1336abc-1b60-46f7-be2d-2105f2daf283"
        }
      ]
      ...
]
```

Under the json path `includes.tracking_scenarios` it will also attach the hole tracking scenarios associated:

```
"includes": {
    ...
    "trackingScenarios": []
    ...
}
```